webui/react/tsconfig.json

```diff
@@ -12,6 +12,7 @@
     "allowSyntheticDefaultImports": true,
     "strict": true,
     "forceConsistentCasingInFileNames": true,
+    "strictPropertyInitialization": false,
```

Comment:
question: What was this addressing?

Reply:
a compiler complaint for the generated swagger code but it's now contained within its own package so this would go away